Quotient/room.cpp

```diff
@@ -1743,7 +1743,7 @@ Room::Private::moveEventsToTimeline(RoomEventsRange events,
         eventsIndex.insert(eId, index);
         if (usesEncryption)
             if (auto* const rme = ti.viewAs<RoomMessageEvent>())
-                if (auto* const content = rme->content())
+                if (const auto content = rme->content())
                     if (auto* const fileInfo = content->fileInfo())
```

Comment:
These two conditions can now be merged into one - GitHub doesn't allow me to make a suggestion with all the de-indentation below, but the idea is:
```suggestion
                if (const auto fileInfo = rme->fileContent())
```